webcompat/views.py

```diff
@@ -51,6 +51,21 @@ def before_request():
 @app.after_request
 def after_request(response):
     session_db.remove()
+    # Security headers
```

Comment:
I would put the full csp thing in a specific function: `add_csp(response)` which returns `response` with the new headers, that I would call here.